datashare-app/src/main/java/org/icij/datashare/WebApp.java

```diff
@@ -48,6 +52,7 @@ static void start(Properties properties) throws Exception {
             waitForServerToBeUp(parseInt(mode.properties().getProperty(PropertiesProvider.TCP_LISTEN_PORT)));
             Desktop.getDesktop().browse(URI.create(new URI("http://localhost:")+mode.properties().getProperty(PropertiesProvider.TCP_LISTEN_PORT)));
         }
+        requeueDatabaseBatches(mode.get(BatchSearchRepository.class), mode.get(TaskManager.class));
```

Comment:
rename to requeueDatabaseBatchSearches ?

Reply:
ok